src/gatling/simulations/simulations/PRL_Simulation.scala

```diff
@@ -150,9 +317,9 @@ class PRL_Simulation extends Simulation {
     simulationType match {
       case "perftest" | "pipeline" => //currently using the same assertions for a performance test and the pipeline
         if (debugMode == "off") {
-          Seq(global.successfulRequests.percent.gte(95),
-            details("XUI_PRL_045_ConfirmPayment").successfulRequests.percent.gte(80),
-            details("PRL_CitizenC100_813_FinalSubmitRedirect3").successfulRequests.percent.gte(80))
+          Seq(global.successfulRequests.percent.gte(95))
+            //details("XUI_PRL_045_ConfirmPayment").successfulRequests.percent.gte(80),
```

Comment:
We should have some transaction percent assertions here still, maybe the last one for each scenario?